### ISSUE TITLE
Cannot read property 'fluid' of null

### ISSUE THREAD

**Issue by mduecker** [cloudinary-devs/gatsby-transformer-cloudinary#3]
Hi Jason,
I just discovered your package and tried out the demo example site.

After running `gatsby develop` I get the following error when trying to load the site:
`Cannot read property 'fluid' of null` 

This is related to the following error during build (gatsby develop):
```
The GraphQL query from /Users/markus/repositories/Bitbucket/gatsby-cloudinary/src/examples/fluid-silly.js failed.

Errors:
  Request failed with status code 400

  GraphQL request:4:7
  4 |       fluid(transformations: ["e_blackwhite"], chained: ["e_vectorize:colors:2:d
    |       ^
    | especkle:20,e_tint:100:tomato:0p:white:100p", "l_beard_png,w_0.77,fl_relative,g_
Plugin:
  none
Query:
  query usersMarkusRepositoriesBitbucketGatsbyCloudinarySrcExamplesFluidSillyJs3123034595 {
    image: file(name: {eq: "marisa"}) {
      cloudinary: childCloudinaryAsset {
        fluid(transformations: ["e_blackwhite"], chained: ["e_vectorize:colors:2:despeckle:20,e_tint:100:tomato:0p:white:100p", "l_beard_png,w_0.77,fl_relative,g_face,a_-5,y_0.06"]) {
          ...CloudinaryAssetFluid
        }
      }
    }
  }

  fragment CloudinaryAssetFluid on CloudinaryAssetFluid {
    aspectRatio
    base64
    sizes
    src
    srcSet
  }
```

**Comment by mduecker** [comment]
I think I found the problem:
`l_beard_png` is the key to an image in your repo?

Then it's no wonder Cloudinary throws an error here.

**Comment by jlengstorf** [comment]
Oh, whoops. Yeah, I didn't actually expect folks to try and run the demo with those images in place. 😕 

I don't think we can demo overlays without referencing images in the account. Named transforms are also a cool feature to highlight that are tied to the account.

I'm not sure what the proper solution is. A disclaimer, maybe?

**Comment by mduecker** [comment]
> I'm not sure what the proper solution is. A disclaimer, maybe?

**Comment by jlengstorf** [comment]
Okay, cool. I'll add this to my Todo list. Thanks!

**Comment by jlengstorf** [comment]
Unless someone wants to get their Hacktoberfest underway?

**Comment by jlengstorf** [comment]
Closed by #7
